hazelcast/src/main/java/com/hazelcast/nio/serialization/GenericRecord.java

```diff
@@ -199,6 +204,64 @@
     @Nullable
     String readUTF(@Nonnull String fieldName);
 
+    /**
+     * Reads a decimal which is arbitrary precision and scale floating-point number to BigDecimal
```

Comment:
nit: trailing dot is missing here and in below.
```suggestion
     * Reads a decimal which is arbitrary precision and scale floating-point number to BigDecimal.
```